### PR TITLE
Bug fix: OMIMPS namespace && no OMIM parents in migration

### PR DESCRIPTION
**Pull request by joeflack4**
Resolves:
- #438
- https://github.com/monarch-initiative/omim/issues/103

## Overview
This PR resolves two things:
1. **OMIMPS namespace**: Was recently changed (https://github.com/monarch-initiative/omim/pull/107#discussion_r1534845966), which addressed https://github.com/monarch-initiative/omim/issues/103. I thought I raised a review comment about this, but either I didn't, or we forgot to follow up with the rest of the tasks in that issue, which leads to the necessity of this change here in `mondo-ingest` to match the updated namespace / URI prefix.
2. **Parents missing in OMIM slurp**: The reason this was happening is because all OMIM parents are in the OMIMPS namespace. However, until addressing (1) in `mondo-ingest` with this PR, those parents could no longer be detected.

### Documentation
Was the documentation added/updated under `docs/`?
- [x] Yes
- [ ] No, updates to the docs were not necessary after careful consideration

### QC
Was the full pipeline run before submitting this PR using `sh run.sh make build-mondo-ingest` on this branch (after 
`docker pull obolibrary/odkfull:dev`), and no errors occurred?
- [x] Yes #479
- [ ] No, there are no functional (code-related) changes to the pipeline in the PR, so no re-run is necessary
   
### New Packages
Were any new *Python packages* added?
- [ ] Yes, [requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full) was updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

Were any other *non-Python packages* added?
- [ ] Yes, [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile) updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

### PR Review and Conversations Resolved
Has the PR been sufficiently reviewed by at least 1 team member of the Mondo Technical team and all threads resolved?
- [ ] Yes

## Related
- https://github.com/monarch-initiative/mondo-ingest/pull/479

### CODE REVIEW

File: src/ontology/metadata/omim.yml [hunk 1]
```diff
@@ -28,4 +28,4 @@ description: >
 homepage: https://www.omim.org/
 base_prefix_map:
   OMIM: https://omim.org/entry/
-  OMIMPS: https://www.omim.org/phenotypicSeries/PS
```
Comment (joeflack4) [inline src/ontology/metadata/omim.yml:31]: ## The primary changes
6 instances of removing `www.` from the OMIMPS URI prefix was done.

Basically I did a global find, found that some files with certain extensions (filter: `!*.owl,!*.ipynb,!*.ttl,!*.tsv`) ought not have the replacement. All other instances I saw seemed like they ought to have the replacement.

The particular change highlighted by this review comment is the one that fixed the missing OMIM slurp parents issue.

File: src/ontology/slurp/omim.tsv [hunk 2]
```diff
@@ -2,25 +2,13 @@ mondo_id	mondo_label	xref	xref_source	original_label	definition	parents
 ID	LABEL	A oboInOwl:hasDbXref	>A oboInOwl:source SPLIT=|		A IAO:0000115	SC %
 MONDO:0958222	maple syrup urine disease,  iia 1b	OMIM:620698	MONDO:equivalentTo	maple syrup urine disease,  iia 1b		
 MONDO:0958223	maple syrup urine disease,  iia 2	OMIM:620699	MONDO:equivalentTo	maple syrup urine disease,  iia 2		
-MONDO:0958224	encephalopathy, porphyria-related	OMIM:620704	MONDO:equivalentTo	encephalopathy, porphyria-related		
-MONDO:0958225	epidermolytic hyperkeratosis 2b, autosomal recessive	OMIM:620707	MONDO:equivalentTo	epidermolytic hyperkeratosis 2b, autosomal recessive		
-MONDO:0958226	leukoencephalopathy, porphyria-related	OMIM:620711	MONDO:equivalentTo	leukoencephalopathy, porphyria-related		
+MONDO:0958225	epidermolytic hyperkeratosis 2b, autosomal recessive	OMIM:620707	MONDO:equivalentTo	epidermolytic hyperkeratosis 2b, autosomal recessive		MONDO:0957316
```
Comment (joeflack4) [inline src/ontology/slurp/omim.tsv:5]: ## Updated slurp output
This isn't really necessary for this PR, and I can undo this change if requested, but I just re-ran `migrate.py` directly for OMIM without updating any prerequisites, etc, and confirmed the fix.

Comment (joeflack4) [inline src/ontology/slurp/omim.tsv:5]: ## QC: `build-mondo-ingest`
I'm currently running the build and will open up a data PR and link it here when it is ready.

**Comment by joeflack4** [comment]
@souzadevinicius FYI: We made a change to remove `www.` from OMIMPS URI prefix and it broke some stuff we had to fix.

File: src/ontology/slurp/omim.tsv (hunk 2 above)
Comment (matentzn) [reply src/ontology/slurp/omim.tsv:5]: Oh wow you already did this! Forget my comments in the OMIM repo then. Good job.

**Review by matentzn — APPROVED** [review]
This looks great! I am very happy with this, and also with the fact that you ran a complete test! Great.

File: src/ontology/slurp/omim.tsv (hunk 2 above)
Comment (joeflack4) [reply src/ontology/slurp/omim.tsv:5]: Sounds good; gonna disregard [this comment](https://github.com/monarch-initiative/omim/issues/103#issuecomment-2026950154) then.. Glad this looks as you would expect.

This is approved, so I am merging.

**Pull request merged**